feat(core): override JSDoc config to customize default documentation (#2084)
orval-labs/orval: docs/src/pages/reference/configuration/output.md

};
```

> #### jsDoc
> 
> Type: `Object`.
> 
> ```ts
> // type signature
> type OverrideJsDocOptions = {
>   filter?: (schema: Record<string, any>) => { key: string; value: string }[];
> };
> ```
> 
> `jsDoc` is a configuration object that allows customizing JSDoc generation by optionally providing a filter function to transform schema entries into key–value pairs.
> 
> Example:

Reviewer: Please prefer to this.

https://orval.dev/reference/configuration/inputhttps://orval.dev/reference/configuration/input

```suggestion
#### jsDoc
##### filter

Type: `Object`.

A configuration object that allows you to customize `JSDoc` generation by optionally providing a filter function that transforms schema entries into key-value pairs.

Example:
```

```ts
module.exports = {
  petstore: {
    output: {
      override: {
        jsDoc: {
          filter: (schema) => {

>             const blacklist = [

Reviewer: This would be more appropriate.

```suggestion
            const allowlist = [
```
